### PR TITLE
John conroy/lazy vitessce

### PR DESCRIPTION
**Pull request by john-conroy**
Dynamically import vitessce visualization component and display circular progress indicator as fallback. At least now first-time visitors won't have to load vitessce when visiting the homepage.

Also fixes issues with visualization title font sizing introduced by #738 and increased bottom margin to prevent overlap. Sorry for the bad code suggestion in one of your earlier PRs, @ilan-gold.

### CODE REVIEW

File: context/app/static/js/components/Detail/Visualization/Visualization.jsx [hunk 1]
```diff
@@ -60,7 +60,9 @@ function Visualization(props) {
   return (
     <StyledSectionContainer id="visualization">
       <StyledHeader>
-        <StyledHeaderText>Visualization</StyledHeaderText>
+        <StyledHeaderText variant="h4" component="h2">
```
Comment (mccalluc) [inline context/app/static/js/components/Detail/Visualization/Visualization.jsx:63]: There was a PR yesterday, I think, that got rid of a lot of these attribute usages, and I'm reluctant to go in the other direction. I've wondered if we could just have an `<H1>`, `<H2>`, that would wrap `<h1>` with whatever material UI details we need.

File: context/app/static/js/components/Detail/VisualizationWrapper/VisualizationWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      fallback={
+        <StyledSectionContainer id="visualization">
+          <StyledHeader>
+            <StyledHeaderText variant="h4" component="h2">
+              Visualization
+            </StyledHeaderText>
+          </StyledHeader>
+          <VisualizationBackground>
+            <CircularProgress />
+          </VisualizationBackground>
```
Comment (mccalluc) [inline context/app/static/js/components/Detail/VisualizationWrapper/VisualizationWrapper.jsx:23]: Good, so the please-wait will use as much height as the final result, right?

Comment (john-conroy) [reply context/app/static/js/components/Detail/VisualizationWrapper/VisualizationWrapper.jsx:23]: Yes.

File: context/app/static/js/components/Detail/Visualization/Visualization.jsx (hunk 1 above)
Comment (john-conroy) [reply context/app/static/js/components/Detail/Visualization/Visualization.jsx]: I think I fixed that, let me know if the change addresses your concerns.

**Pull request merged**